src/Sylius/Component/Promotion/Repository/PromotionCouponRepositoryInterface.php

```diff
@@ -32,4 +33,12 @@ public function createQueryBuilderByPromotionId($promotionId);
      * @return int
      */
     public function countByCodeLength($codeLength);
+
+    /**
+     * @param string $promotionCode
+     * @param string $code
+     *
+     * @return PromotionInterface
+     */
+    public function findOneByPromotionCodeAndCode($promotionCode, $code);
```

Comment:
It's done this way because `$promotionCode` is the first argument, `$code` is the second one. I just wanted to be consistent with arguments order (which IMO is a good reason to make this function slightly worse named).